ls.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# !source env/bin/activate
+
+import argparse
+import os
+import sys
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-a", "--all", action='store_true')
+
+args = parser.parse_args()
+
+if args.all:
+    sys.stdout("\n".join(os.listdir()))
+else:
+    sys.stdoutl("Oops")
```

Comment:
1. Здесь опечатка
2. `stdout` это не функция, а по сути файл в который нужно делать `write`. `sys.stdout.write("Oops")`
```suggestion
    sys.stdout("Oops")
```

Reply:
Из-за этого скрипт просто падает